grpc/weaviate.proto

```diff
@@ -158,6 +159,19 @@ message HybridSearchParams {
   FusionType fusion_type = 5;
 }
 
+message NearTextSearchParams{
+  message Move{
+    float force =1;
+    repeated string concepts = 2;
+    repeated string uuids = 3;
+  }
+
+  repeated string queries = 1;
```

Comment:
what are queries? shouldn't be this `repeated string concepts`?

Reply:
After some discussion we decided to rename this from `concepts` to query. I always found that word unfitting - you're not looking for a concept, you're looking for something that is close to the text/query that you are providing.

For the Move* parameter, we decided to stay with concepts. While it also just a piece of text, it feels similar right to use here. You are searching for a *text*, but want to move away from this concept